osclib/stagingapi.py

```diff
@@ -144,9 +144,9 @@ def is_staging_manager(self):
             self._is_staging_manager = self.is_user_member_of(self.user, self.cstaging_group)
         return self._is_staging_manager
 
-    def project_has_repo(self, repo_name):
+    def project_has_repo(self, repo_name, project=None):
```

Comment:
I think this is unnecessary, `self.project` is the main project already

Reply:
on "usual" projects it would have worked great, but SUSE:SLFO:Main unfortunately doesn't have any repo defined as the contents are built inside SUSE:SLFO:Main:Build, so we can only check on another project (like the staging itself)